### PR TITLE
Reworked operational pausing

### PR DESCRIPTION
**Pull request by drinkcoffee**

### CODE REVIEW

File: docs/framework/20categories/40operation/ability-pause.md [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+is unlikely to effective, as by this time, the project may have already been drained of funds.
+
+To prevent malicious parties unnecessarily halting the project by calling the ```pause()``` function, ```pause()``` functions need to have some form of [access control](../30implementation/access-control.md). 
+
+For projects operated by a single organization, a simple administration set-up could be used. However, as described below, this has serious issues. For example, a single administrator account (that is an *owner* account) could be the only party authorized to call the ```pause()``` function. However, to be responsive to the need to pause the project, the private key belonging to the administrator account would need to be shared with support staff who live in time zones around the world. The issue with sharing a single private key is that it would be impossible to determine which support staff member used the key to pause the project. Additionally, if one of the support staff left the company or were compromised, then the shared key would need to be changed.
```
Comment (ermyas) [inline docs/framework/20categories/40operation/ability-pause.md:8]: nit: consistency in using either "time zone", "timezone", or "time-zone"

File: docs/framework/20categories/30implementation/pause.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 ### Ability to Pause Project
```
Comment (ermyas) [inline docs/framework/20categories/30implementation/pause.md:1]: The current write-up assumes that pausing is only a manual operation. Do we have thoughts on pausing being performed automatically by systems that monitor for anomalies? This obviates most of the timezone considerations discussed below.

File: docs/framework/20categories/40operation/ability-pause.md [hunk 3]
```diff
@@ -1,4 +1,21 @@
...
+
+An better approach to using a shared key is to provide each support staff member with their own administrator account, and provide all accounts with the *Pauser Role*. The advantage of this approach over using a shared key is that the account that pauses the project can be associated with a specific support staff member. Additionally, if one of the support staff left the company or were compromised, then only their account would need to be disabled. This would allow the other support staff to operate without changing their keys.
+
+For projects that are operated by multiple organizations the pausing capability should be controlled by a [multi-signature wallet](https://medium.com/gauntlet-networks/multisig-transactions-with-gnosis-safe-f5dbe67c1c2d). Multi-signature wallets have a threshold number of owners that must vote for a proposed transaction. In the case of pausing, the proposed transaction would be to call the ```pause()``` function.
```
Comment (ermyas) [inline docs/framework/20categories/40operation/ability-pause.md:12]: What are our thoughts on pausing capabilities issued to privileged stakeholders that are not operators of the bridge?

**Review by ermyas — APPROVED** [review]

File: docs/framework/20categories/40operation/ability-pause.md [hunk 4]
```diff
@@ -1,4 +1,21 @@
 ### Ability to Pause
-If pausing is controlled by a multi-sig, then there is the risk 
-that not enough parties can be gathered quickly enough to pause 
-the project.
+The pausing capability described in the [Ability to Pause Implementation Risk section](../30implementation/pause.md)
+is only effective if the ```pause()``` function can be executed expeditiously. Typically, attacks on protocols are mounted in time periods ranging from minutes to hours. As such, pausing a project several hours after an attack has commenced 
+is unlikely to effective, as by this time, the project may have already been drained of funds.
```
[the remark this answers is not in the document]
Comment (drinkcoffee) [reply docs/framework/20categories/40operation/ability-pause.md:4]: Correct

File: docs/framework/20categories/40operation/ability-pause.md [hunk 5]
```diff
@@ -1,4 +1,21 @@
...
+
+An better approach to using a shared key is to provide each support staff member with their own administrator account, and provide all accounts with the *Pauser Role*. The advantage of this approach over using a shared key is that the account that pauses the project can be associated with a specific support staff member. Additionally, if one of the support staff left the company or were compromised, then only their account would need to be disabled. This would allow the other support staff to operate without changing their keys.
```
[the remark this answers is not in the document]
Comment (drinkcoffee) [reply docs/framework/20categories/40operation/ability-pause.md:10]: Fixed

File: docs/framework/20categories/40operation/ability-pause.md (hunk 3 above)
Comment (drinkcoffee) [reply docs/framework/20categories/40operation/ability-pause.md:12]: I expect you mean LPs being given permission to pause the system? Does that ever happen?

File: docs/framework/20categories/40operation/ability-pause.md (hunk 1 above)
Comment (drinkcoffee) [reply docs/framework/20categories/40operation/ability-pause.md]: Going for one word.

File: docs/framework/20categories/40operation/ability-pause.md (hunk 3 above)
Comment (ermyas) [reply docs/framework/20categories/40operation/ability-pause.md]: Our stakeholders section distinguishes between bridge operators and bridge validators. I gather the current writeup focuses on operators, but we might want to discuss if we assume pausing capability can/should be granted to validators as well (e..g third-party attestors).

Comment (drinkcoffee) [reply docs/framework/20categories/40operation/ability-pause.md]: Thanks for pointing out that separation we have in the stakeholders section. I will add in a pragraph about this.

**Pull request merged**